src/js/api.js

```diff
@@ -101,59 +84,6 @@ var api = {
 
         watch('environment', onChange);
     },
-
-    nonWebWorkerRequest: function() {
```

Comment:
We probably want to remove the references to data we no longer want to display rather than remove the entire function as it is called by subscribeToDataUpdates().